Formula/bat.rb

```diff
@@ -19,13 +19,8 @@ def install
     ENV["SHELL_COMPLETIONS_DIR"] = buildpath
     system "cargo", "install", "--locked", "--root", prefix, "--path", "."
 
-    # In https://github.com/sharkdp/bat/pull/673,
-    # documentation and fish autocompletion got parameterized
-    inreplace "assets/manual/bat.1.in", "{{PROJECT_EXECUTABLE | upcase}}", "bat"
-    inreplace "assets/manual/bat.1.in", "{{PROJECT_EXECUTABLE}}", "bat"
-    inreplace "assets/completions/bat.fish.in", "{{PROJECT_EXECUTABLE}}", "bat"
-    man1.install "assets/manual/bat.1.in" => "bat.1"
-    fish_completion.install "assets/completions/bat.fish.in" => "bat.fish"
+    man1.install "target/release/build/bat-df82c97c3bd64ddc/out/assets/manual/bat.1"
+    fish_completion.install "target/release/build/bat-df82c97c3bd64ddc/out/assets/completions/bat.fish"
```

Comment:
`/bat-df82c97c3bd64ddc/` looks a bit suspicious

Reply:
Yep, it’s not ideal. While I’m not sure where its value comes from, I can confirm that on two separate clean runs of the `cargo install` process, using the v0.13.0 archive, it generated the same value.

So it is, I think, internally consistent, but probably changes version-by-version.

Reply:
I'm also not sure where it comes from, but I would suggest to use a glob with a wildcard here (`target/release/build/bat-*/out/…`). There will be only one `bat` folder. We do something similar for other distributions as well.

Reply:
@sharkdp yep, that’s what 71fe52f does, so we should be all set!